client/state/selectors/is-connected-secondary-network-site.js

```diff
@@ -0,0 +1,43 @@
+/**
+ * External dependencies
+ */
+import { get } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import createSelector from 'lib/create-selector';
+import { isMainSiteOf } from 'state/selectors';
+import { isJetpackSiteSecondaryNetworkSite } from 'state/sites/selectors';
+
+/**
```

Comment:
Hi @tyxla , you are right, with the comments it was  not possible to understand the difference of the selectors. So I added a clarification in the comments to make things easier. Thank you for the suggestion.